reports-scheduler/src/main/kotlin/com/amazon/opendistroforelasticsearch/reportsscheduler/model/RestTag.kt

```diff
@@ -38,6 +38,8 @@ internal object RestTag {
     const val END_TIME_FIELD = "endTimeMs"
     const val REPORT_DEFINITION_FIELD = "reportDefinition"
     const val REPORT_DEFINITION_ID_FIELD = "reportDefinitionId"
+    const val STATS_START_TIME = "startTime"
```

Comment:
Please also keep unit name for the time.

Reply:
Not using **STATS_START_TIME** and **STATS_END_TIME**, will remove this.